src/listenMqtt.js

```diff
@@ -502,6 +502,10 @@ module.exports = function (defaultFuncs, api, ctx) {
   return function (callback) {
     globalCallback = callback;
 
+    //Reset some stuff
+    lastSeqId = 0;
```

Comment:
Should these also be in ctx?

Reply:
They are mqtt specific, So I just left them here and reset them when listenMqtt is called and they worked fine.

Reply:
I can't imagine the mqtt clients are supposed to share this information though

Reply:
The use case that we're fixing is people who are logging into multiple accounts at the same time.
In this case, there would be one ctx object per "api" object, so if you call listenMqtt on account2 after calling it on account1, it won't reset the values for account1.

If we store the variables globally at the top level of this file, all instances of the api will need to share the values. We don't want any interaction at all between instances of the api.

Reply:
Alright. Sorry for misunderstanding 😁
I'll change it as soon as I can.